src/cms-content/about/index.tsx

```diff
@@ -30,8 +30,6 @@ interface AboutContent {
   partnersContent: PartnersContent[];
   teamHeader: string;
   teamIntro: Markdown;
-  futureProjectsHeader: string;
-  futureProjectsContent: Markdown;
```

Comment:
We should remove these from the CMS config too if we no longer support them: https://github.com/covid-projections/covid-projections/blob/develop/public/admin/config.yml#L98